soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -4266,6 +4387,19 @@ void DrawRandoEditor(bool& open) {
 
                 UIWidgets::PaddedSeparator();
 
+                // Shuffle Merchants
+                ImGui::Text(Settings::ShuffleMerchants.GetName().c_str());
+                UIWidgets::InsertHelpHoverText(
+                    "Enabling this adds a Giant's Knife and a pack of Bombchus to the item pool "
+                    "and changes both Medigoron and the Haunted Wasteland Carpet Salesman to sell "
+                    "a random item once at the price of 200 rupees.\n\n"
+                    "On (no hints) - Salesmen will be included but won't tell you what you'll get.\n"
+                    "On (with hints) - Salesmen will be included and you'll know what you're buying."
```

Comment:
This is probably fine for now, but eventually I think we'd want the hints/no hints to be a separate option near the gossip stone settings.

Reply:
That was originally what I went for, as 3d rando also had the description of the checks dictated by hint clarity. However, I spoke with @leggettc18 and there isn't obscure and vague text for the checks if i recall correctly. So, it's set to clear/super vague/off for now. I did add some logic to hint that it might be foolish to purchase but that can also be removed on request.